Plugin.pm

```diff
@@ -136,7 +136,11 @@ sub handleFeed {
 		name => cstring($client, 'PLAYLISTS'),
 		image => 'html/images/playlists.png',
 		type => 'link',
-		url => \&getFavoritePlaylists,
+		# I don't think that notion still exists. it seems to me that the
```

Comment:
If `getFavorites` indeed returns the user's own playlists, then there's no need for this any more. Please remove it completely if unused.

Reply:
I'll wait a bit to see if anything comes back 😄